pkg/kubelet/lifecycle/handlers.go

```diff
@@ -133,7 +137,7 @@ func getHttpRespBody(resp *http.Response) string {
 		return ""
 	}
 	defer resp.Body.Close()
-	if bytes, err := ioutil.ReadAll(resp.Body); err == nil {
+	if bytes, err := sysutils.ReadAllWithLimit(resp.Body, maxReadLength); err == nil {
```

Comment:
AFAICT, the body is only used to include a message for debug purposes, so I think this is an example of a case where we would want to ignore the readlimit error.

I would also lower the read limit for this. Maybe 10K?

Reply:
Fixed